quiz.go

```diff
@@ -7,9 +7,15 @@ import (
 	"io"
 	"log"
 	"os"
+	"time"
 )
 
+func printResults(numCorrect int, numIncorrect int) {
```

Comment:
One other thing you could consider here is a stats struct and then implementing the [Stringer interface](https://golang.org/pkg/fmt/#Stringer).

```go
func (s *Stats) String() string{
    return fmt.Sprintf("Correct: %v, incorrect: %v\n", s.correct, s.incorrect)
}
```